### PR TITLE
Made changes to closures for coding conventions (Issue #127)

### PR DESCRIPTION
**Pull request by kazoni**

### CODE REVIEW

**Comment by raj454raj** [comment]
Issue #127

**Comment by kazoni** [comment]
Restored appadmin, I looked through all the .py files and changed the ones I saw.  There really weren't too many.

File: controllers/default.py [hunk 1]
```diff
@@ -1101,7 +1101,7 @@ def download_submission():
 
     from bs4 import BeautifulSoup
 
-    def handle_retrieve_error(download_url,
+    def _handle_retrieve_error(download_url,
                               status_code,
```
Comment (raj454raj) [inline controllers/default.py]: Oops you need to indent this too (1 more space for each argument)

**Comment by raj454raj** [comment]
@kazoni : Just indent that function args and squash the 3 commits into 1 and it will be merged after that.

File: controllers/default.py [hunk 2]
```diff
@@ -1142,50 +1142,50 @@ def response_handler(download_url, response):
         """
 
         if response.status_code != 200:
-            return handle_retrieve_error(download_url,
+            return _handle_retrieve_error(download_url,
                                          response.status_code)
```
Comment (raj454raj) [inline controllers/default.py]: Here too

File: controllers/default.py [hunk 3]
```diff
@@ -1142,50 +1142,50 @@ def response_handler(download_url, response):
...
 
         try:
             return BeautifulSoup(response.text, "lxml").find("pre").text
         except Exception as e:
-            return handle_retrieve_error(download_url,
+            return _handle_retrieve_error(download_url,
                                          response.status_code,
```
Comment (raj454raj) [inline controllers/default.py]: Here too

**Comment by kazoni** [comment]
All fixed.  I didn't even think of aligning the arguments.

**Pull request merged**